lib/sharezone_widgets/lib/src/theme/brightness/dark_theme.dart

```diff
@@ -51,6 +51,52 @@ final darkTheme = ThemeData(
   colorScheme: ColorScheme.fromSwatch()
       .copyWith(secondary: _accentColor, brightness: Brightness.dark),
   tabBarTheme: TabBarTheme(labelColor: Colors.white),
+  checkboxTheme: CheckboxThemeData(
+    fillColor:
+        MaterialStateProperty.resolveWith<Color>((Set<MaterialState> states) {
+      if (states.contains(MaterialState.disabled)) {
+        return null;
+      }
+      if (states.contains(MaterialState.selected)) {
+        return _accentColor;
+      }
+      return null;
```

Comment:
For what are these conditions?

Reply:
It was automatically generated. I think so that the theme is the same before and after the update because otherwise the theme might work differently now than to before? Not sure.